PizzaOvenUI/main.qml

```diff
@@ -301,7 +301,8 @@ Window {
 
         if (currentScreen === newScreenName)
         {
-            console.log("Transition to same screen, so don't switch.")
+            console.log("Transition to same screen, so don't switch. Attempt to update the screen.")
+            stackView.currentItem.update();
```

Comment:
This is the other half of the screen update when a new cycle is loaded from the app while a previous loading is being displayed.